utility/pigpio/compatibility.h

```diff
@@ -1,23 +1,16 @@
 /*
- * File:   compatiblity.h
- * Author: purinda
- *
- * Created on 24 June 2012, 3:08 PM
- * patch for safer monotonic clock & millis() correction for 64bit LDV 2018
+ * Time keeping functions
  */
 
-#ifndef COMPATIBLITY_H
-#define COMPATIBLITY_H
+#include <stdint.h> // for uintXX_t types
+
+#ifndef RF24_UTILITY_PIGPIO_COMPATIBLITY_H_
+#define RF24_UTILITY_PIGPIO_COMPATIBLITY_H_
```

Comment:
this suggestion is not warranted. clang-format is confused about the header guards in this file because there is another pre-processor statement before this... Header guards should be the first pre-processor statements in the file.